scripts/bytecodecompare/prepare_report.js

```diff
@@ -110,7 +110,7 @@ for (const optimize of [false, true])
                         if ('metadata' in contractResults && cleanString(contractResults.metadata) !== undefined)
                             metadata = contractResults.metadata
 
-                        console.log(filename + ':' + contractName + ' ' + bytecode)
+                        console.log('BYTECODE COMPARE TEST ' + filename + ':' + contractName + ' ' + bytecode)
```

Comment:
Right, I just wanted to ask whether you did something like this :-).

Reply:
Yeah. Still waiting to see the results.

As for a successful run, you can see how it looks like here: https://app.circleci.com/pipelines/github/ethereum/solidity/15012/workflows/46282180-d244-4043-8474-d2b22f1c9394/jobs/684330/steps

Reply:
Looks like the negative case works fine too: https://app.circleci.com/pipelines/github/ethereum/solidity/15019/workflows/488b11bd-b8f9-414b-8457-a1483f42fed4/jobs/684593/steps